arangod/Cluster/ClusterInfo.cpp

```diff
@@ -1511,16 +1555,28 @@ void ClusterInfo::loadPlan() {
   }
   TRI_IF_FAILURE("AlwaysSwapAnalyzersRevision") { swapAnalyzers = true; }
 
-  decltype(_replicatedLogs) newReplicatedLogs;
-  decltype(_collectionGroups) newCollectionGroups;
+  decltype(_replicatedLogs) newReplicatedLogs{_resourceMonitor};
+  decltype(_collectionGroups) newCollectionGroups{_resourceMonitor};
+
+  static_assert(std::uses_allocator_v<
+                ClusterInfo::NewStuffByDatabase,
+                ResourceUsageAllocator<ClusterInfo::NewStuffByDatabase>>);
+
+  std::make_obj_using_allocator<NewStuffByDatabase>(
+      ResourceUsageAllocator<int>(_resourceMonitor));
```

Comment:
My understanding of this statement is that it creases a temporary `NewStuffByDatabase` object on the stack, which, because it is anonymous, is not referred to later.

Reply:
left over, I will remove this.